pom.xml

```diff
@@ -142,6 +142,58 @@
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>io.confluent</groupId>
+                <version>0.9.0-SNAPSHOT</version>
+                <artifactId>kafka-connect-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>kafka-connect</goal>
+                        </goals>
+                        <configuration>
+                            <title>Kafka Connect JDBC</title>
```

Comment:
If we don't include a title, doesn't it take the title from the POM? Why duplicate this here if it's the same?